src/main/java/com/textmessenger/model/User.java

```diff
@@ -34,6 +35,7 @@
   private String email;
 
   @Column(name = "password")
+  @JsonIgnore
```

Comment:
This will be handled on a DTO level

Reply:
User is not a DTO level?

Reply:
No, User is Entity level. Specific DTO will be copy of User, used to communicate with front-end. Discussed today with @podolsky4

Reply:
we talk about @projector ?

Reply:
No, we don't need automatic repositories with projections now